src/ixdat/plotters/tpms_plotter.py

```diff
@@ -56,27 +61,36 @@ def plot_measurement(
                 masses in which case one list is plotted on the left y-axis and the other
                 on the right y-axis of the top panel.
             mol_list (list of str): The names of the molecules, eg. ["H2", ...] to
-                plot. Defaults to all of them (measurement.mass_list)
+                plot. Defaults to all of them if quantified (measurement.mass_list)
             mol_lists (list of list of str): Alternately, two lists can be given for
                 molecules in which case one list is plotted on the left y-axis and the
                 other on the right y-axis of the top panel.
             tspan (iter of float): The time interval to plot, wrt measurement.tstamp
             tspan_bg (timespan): A timespan for which to assume the signal is at its
                 background. The average signals during this timespan are subtracted.
                 If `mass_lists` are given rather than a single `mass_list`, `tspan_bg`
-                must also be two timespans - one for each axis. Default is `None` for no
+                can also be two timespans - one for each axis. Default is `None` for no
                 background subtraction.
             remove_background (bool): Whether otherwise to subtract pre-determined
                 background signals if available. Defaults to (not logplot)
             unit (str): the unit for the MS data. Defaults to "A" for Ampere
+            x_unit (str): unit for x axis defaults to 's' for seconds
+            meta_units (dict): dictionary with new units for y axis on bottom panel.
+                Default to ValueSeries.unit.name for last plotted meta_name
+            meta_list (list of str): The names of the data_series plotted on axes[1].
+                Defulats None.
```

Comment:
I still don't like this, I have to say. "Defaults None" doesn't tell the user about how the method *behaves*, because what gets plotted by default is T on the lower left y-axis and P on the lower right y-axis. 
I think the point of this is that sometimes the user might want to plot more than one temperature or pressure, right? If so, it would be more transparent and intuitive to have optional arguments called `T_name_list` and `P_name_list` which could be used as an alternative to `T_name` and `P_name`. 
I think this would make some of the code more readable too.

Reply:
I have changed the meta_list to T_names and P_names. The readability I still think is poor. Unfortunately.

Reply:
Leaving this open because I may give a try simplifying later.

Reply:
Ah, never mind, now that I am looking at the new code: I like it! It looks better to me.